yellowbrick/text/freqdist.py

```diff
@@ -67,15 +80,20 @@ def freqdist(X, y=None, ax=None, n=50, orient="h", color=None, **kwargs):
         Returns the fitted, finalized visualizer
     """
     # Instantiate the visualizer
-    visualizer = FreqDistVisualizer(ax=ax, n=n, orient=orient, color=color, **kwargs)
+    viz = FreqDistVisualizer(features, ax=ax, n=n, orient=orient, color=color, **kwargs)
```

Comment:
Thank you very much for correcting the quick method function params to match the class params!